docs/manual/index-checker.tex

```diff
@@ -457,8 +460,65 @@ \section{Binary search indices\label{index-searchindex}}
   This is the top type, and programmers should rarely need to write it.
 \end{description}
 
+\section{IndexOf indices\label{index-indexof}}
+
+The sole purpose of the IndexOf Checker is to allow expressing the
+properties of the return value of the JDK's
+\sunjavadoc{java/lang/String.html\#indexOf-java.lang.String-}{String.indexOf} and
+\sunjavadoc{java/lang/String.html\#lastIndexOf-java.lang.String-}{String.lastIndexOf} methods.
+These methods return an index of a substring, or -1 if the substring is not found.
+
+Because of the value -1, the return type of these methods cannot be
+annotated by a \<@LTLengthOf> with a positive offset.
+The \<@IndexOfIndexFor> annotation solves it by having the same
+meaning as \<@LTLengthOf> with the same parameters, except additionally
```

Comment:
This is the most important information in this entire section, and it should be the most prominent.
The description is actually incorrect, since `@LTLengthOf` already permits -1.  Did you mean `@IndexFor`?